private/buf/bufcli/remote_discovery_test.go

```diff
@@ -12,89 +12,95 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package bufmoduleref
+package bufcli
```

Comment:
Should this be rolled into the existing `bufcli_test.go`?

Reply:
Yup perfect, renamed the pkg, although in that file you mention we do openbox `bufcli` testing 😞

Reply:
Also, moved the other tests to closed box, and moved everything to the same file.